spec/requests/api/v1/schedules_spec.rb

```diff
@@ -0,0 +1,138 @@
+require 'swagger_helper'
+
+RSpec.describe 'api/v1/schedules', type: :request do
+  before(:each) do
+    @user = User.create(username: 'juan', email: 'email@hotmail.com', password: '153624')
+    @city = City.create(city_name: 'Brasil')
+    @teacher = Teacher.create(name: 'Carlos', last_name: 'perez', degree: 'mastery')
```

Comment:
I get this error: 
![image](https://user-images.githubusercontent.com/31193823/225813917-7d0b3e6f-fc9e-430c-8f85-ff238a8759e6.png)

I get this error whenever the Teacher model is referenced in tests. If you'd need further help with identifying the issue, feel free to reach out to me and may be able to give you 15 minutes to help solve. Can reach out through slack or email. (Details in github profile)